validation/src/test/scala/hmda/validation/rules/lar/validity/V300Spec.scala

```diff
@@ -0,0 +1,82 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Geography, LoanApplicationRegister }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+class V300Spec extends LarEditCheckSpec {
+
+  property("Succeeds for valid combination of MSA, state, county and tract when MSA != NA") {
+    forAll(larGen) { lar =>
+      val validGeography = Geography("17020", "06", "007", "0036.00")
+      val validLar = lar.copy(geography = validGeography)
+      validLar.mustPass
+    }
+  }
+
+  property("Succeeds for valid combination of MD, state, county and tract when MSA != NA") {
+    forAll(larGen) { lar =>
+      val validGeography = Geography("14454", "25", "025", "0001.00")
+      val validLar = lar.copy(geography = validGeography)
+      validLar.mustPass
+    }
+  }
+
+  property("Succeeds for valid combination of MSA, state, county and tract when MSA == NA") {
+    forAll(larGen) { lar =>
+      val validGeography = Geography("NA", "06", "007", "0036.00")
+      val validLar = lar.copy(geography = validGeography)
+      validLar.mustPass
+    }
+  }
+
+  property("Succeeds for valid combination of MD, state, county and tract when MSA == NA") {
+    forAll(larGen) { lar =>
+      val validGeography = Geography("NA", "25", "025", "0001.00")
+      val validLar = lar.copy(geography = validGeography)
+      validLar.mustPass
+    }
+  }
+
+  property("Succeeds for MD, state, county and tract all = NA") {
+    forAll(larGen) { lar =>
+      val validGeography = Geography("NA", "NA", "NA", "NA")
+      val validLar = lar.copy(geography = validGeography)
+      validLar.mustPass
+    }
+  }
+
+  property("Fails for valid MSA not NA in a small county") {
```

Comment:
I meant to say that it is a valid combination of fields but tract is not NA and the county is small one. I'll fix the language.